src/passes/Inlining.cpp

```diff
@@ -1398,7 +1398,8 @@ struct Inlining : public Pass {
           continue;
         }
         Name inlinedName = inlinedFunction->name;
-        if (!isUnderSizeLimit(func->name, inlinedName)) {
+        if (!isUnderSizeLimit(
+              func->name, inlinedName, getPassRunner()->options)) {
```

Comment:
It might be simpler to not pass in the options, and call `getPassRunner()` in the called function?

Reply:
yes. i reverted this part. thank you.